modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/utils/utils.es.js

```diff
@@ -59,6 +59,17 @@ export function deleteCache() {
 	client.cache.clear();
 }
 
+export function deleteCacheKey(query, variables) {
+	const keyObj = {
+		operation: {
+			query,
+			variables,
+		},
+	};
+	keyObj.fetchOptions = {};
```

Comment:
This is left outside the object because the SF fails because wants the alphabetical order of the properties, but the key generation that is used inside the graphql cache takes the order into account and needs the fetchOptions to be after the operation property to make a match with the existing query in the cache

Reply:
👍

Reply:
@javierdearcos if it is an eslint rule you can try disabling it with an inline comment. This may come from portals SF though and we don't have that benefit. Could we add a comment explaining why its not in the object, otherwise I could imagine a hasty refactor if someone comes across this bit of code later.

Reply:
Good point 🙂 

I'll try to disable the rule as you said or leave a comment otherwise explaining this weird hack